src/constants.ts

```diff
@@ -0,0 +1,38 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { SpecificationFile } from './hooks/validation';
+import { Context } from './hooks/context/models';
+
+export const CONTEXTFILE_PATH = path.resolve(os.homedir(), '.asyncapi');
+
+export class TestingVariables {
```

Comment:
For me TestingVariables is not such a good name. I will name it something like `ContextFixtures`

Fixtures is that kind of background data that you need on a test in order to make it works. That's why I think that this name could be better 😅

Reply:
just have test in name so it is clear what that's for